test/spec/openapi/refs.js

```diff
@@ -282,3 +282,155 @@ test('uses examples if has property required in body', async (t) => {
   t.ok(schema.parameters)
   t.same(schema.parameters[0].in, 'query')
 })
+
+test('support absolute refs in schema', async (t) => {
```

Comment:
These due tests:

- `support absolute refs in schema`
- `support relative refs in schema`

Are already supported by json-schema-resolver https://github.com/Eomm/json-schema-resolver/pull/8

The `openapi` conversion instead ( test named `support definitions keyword in schema`) is provided by this module.

So the questions are:

- is this an issue with the conversion from swagger to openapi?
- is this an issue with `fast-json-stringify`? (as it resolves the schemas by its own)

So, right now, I think this fix does not solve the core problem

Reply:
Thanks for the detailed information, I need to catch up on this 👍🏽 
I want to explore a bit the openapi repo as it is source of best practices in these cases